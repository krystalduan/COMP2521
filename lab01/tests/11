45
-5
11
-7