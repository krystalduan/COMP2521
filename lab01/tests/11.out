-7
-5
11
45
